metrics/ci_integrations/test/source/github_actions/adapter/github_actions_source_client_adapter_test.dart

```diff
@@ -1064,6 +1064,156 @@ void main() {
       },
     );
 
+    test(
+      ".fetchBuildsAfter() maps fetched run jobs' startedAt date to the completedAt date if the startedAt date is null",
+      () async {
+        final completedAt = DateTime.now();
+        final workflowRun = WorkflowRun(number: 2);
+        final workflowRunsPage = WorkflowRunsPage(values: [workflowRun]);
+        final workflowRunJob = WorkflowRunJob(
+          name: jobName,
+          startedAt: null,
+          completedAt: completedAt,
+        );
+
+        whenFetchWorkflowRuns(withArtifactsPage: defaultArtifactsPage)
+            .thenSuccessWith(workflowRunsPage);
+        when(githubActionsClientMock.fetchRunJobs(
+          any,
+          status: anyNamed('status'),
+          page: anyNamed('page'),
+          perPage: anyNamed('perPage'),
+        )).thenSuccessWith(
+          WorkflowRunJobsPage(values: [workflowRunJob]),
+        );
+
+        final firstBuild = testData.generateBuildData(buildNumber: 1);
+
+        final result = await adapter.fetchBuildsAfter(jobName, firstBuild);
+
+        expect(result.first.startedAt, equals(completedAt));
```

Comment:
```suggestion
        final startedAt = result.first.startedAt;
        
        expect(startedAt, equals(completedAt));
```
Would it be more readable? Please apply to the below test cases as well.